packages/create-liveblocks-app/bin/templates/example/example-template.ts

```diff
@@ -85,17 +85,27 @@ export async function create(flags: Record<string, any>) {
     });
 
     if (vercelData.repo) {
-      vercelSpinner.text = c.whiteBright.bold("Cloning new repo...");
+      vercelSpinner.text = "Cloning new repo...";
```

Comment:
Very nitpicky but can we please use the proper ellipsis character instead? Here it is to make it easy for you `…`.